packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -113,6 +115,8 @@ class UpgradeCommandRunner {
     } else {
       await runCommandSecondHalf(flutterVersion);
     }
+    final Duration execution = timer.elapsed;
+    globals.printStatus('Took ${execution.inMinutes} minutes');
```

Comment:
This will print 'Took 0 minutes' if it takes less than a minute to upgrade. Can we add seconds as well (e.g., 'Completed in 3:15.0')?

Reply:
If you add this, consider adding a `elapsedAsMinutes` helper here: https://github.com/flutter/flutter/blob/17848992dc09517fc8975b54d5d831802730665a/packages/flutter_tools/lib/src/base/utils.dart#L90-L100